### DISCUSSION TITLE
remove functionalities that require freemode script control

### DISCUSSION THREAD

**Discussion by lonelybud** [YimMenu/YimMenu#3264] [General]
they seem to break freemode script for example someone entering inside an interior etc. 

The last time it happened when someone was trying to enter garage and a modder tried to kick me - OOM Kick, End Session Kick,  Null Function Kick (in order). He got kicked instead bcz i was host. The person who was trying to enter garage got stuck afterwards.

**Comment by Rxann** [comment]
It will be fixed soon most likely
